index.bs

```diff
@@ -3016,6 +3016,10 @@ When verifying a given {{PublicKeyCredential}} structure (|credential|) and an {
 1. Using the credential public key looked up in step 3, verify that |sig| is a valid signature over the binary concatenation of
     |aData| and |hash|.
 
+    Note: This verification step is automatically compatible with signatures generated by FIDO U2F authenticators, because the
```

Comment:
re (from https://github.com/w3c/webauthn/pull/850#issue-176761510):
> Should we move this to a small subsection of §6.1 Authenticator Data and refer the note to there?

that is fine by me.  So I spect this note would become something like:

`Note: This verification step is automatically compatible with signatures generated by FIDO U2F authenticators, see [[#sctn-fido-u2f-sig-format-compat]]`

Reply:
Done, please re-review!